invenio_rdm_records/services/services.py

```diff
@@ -129,7 +129,7 @@ def search_community_records(
     ):
         """Search for records published in the given community."""
         self.require_permission(identity, "read")
-        current_communities.service.record_cls.pid.resolve(
+        community = current_communities.service.record_cls.pid.resolve(
```

Comment:
note for reviewers: the resolver is able to treat both slugs and uuids, see https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/communities/records/systemfields/pidslug.py#L31